### PR TITLE
Adding build.ps1 scripts

### PR DESCRIPTION
**Pull request by rapatchi**
This is to address #21

### CODE REVIEW

File: Actors/CounterActor/build.ps1 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (ashishnegi) [inline Actors/CounterActor/build.ps1:1]: Extra line at top of file.

File: Actors/CalculatorActor/build.ps1 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+pushd src\CalculatorActorApplication\CalculatorActor.Interfaces\
```
Comment (ashishnegi) [inline Actors/CalculatorActor/build.ps1:1]: Does the script stop if any one command fail in the script ?

File: Actors/CounterActor/build.ps1 (hunk 1 above)
Comment (rapatchi) [reply Actors/CounterActor/build.ps1:1]: removed the extra line

File: Actors/CalculatorActor/build.ps1 (hunk 2 above)
Comment (rapatchi) [reply Actors/CalculatorActor/build.ps1:1]: No it currently does not do that.

**Pull request merged**